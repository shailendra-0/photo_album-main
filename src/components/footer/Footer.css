.Footer {
  height: 20vh;
  font-size: larger;
  color: #fff !important;
  background-color: #2196f3 !important;
  display: flex;
  align-items: center;
  justify-content: center;
}
