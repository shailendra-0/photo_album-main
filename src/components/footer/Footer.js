import React from "react";
import "./Footer.css";

const Footer = () => {
  return <div className="Footer">Â© All rights reserved to Debojeet Jha</div>;
};

export default Footer;
