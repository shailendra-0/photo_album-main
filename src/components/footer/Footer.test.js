import Footer from "./Footer";
import { render, screen } from "@testing-library/react";

it("renders Navbar snapshot", () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment(<Footer />)).toMatchSnapshot();
});

it("renders copyright text in footer", () => {
  render(<Footer />);
  const element = screen.getByText(/Â© All rights reserved to Debojeet Jha/i)
  expect(element).toBeTruthy()
})