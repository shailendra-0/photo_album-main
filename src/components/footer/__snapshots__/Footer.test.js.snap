// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navbar snapshot 1`] = `
<DocumentFragment>
  <div
    class="Footer"
  >
    Â© All rights reserved to Debojeet Jha
  </div>
</DocumentFragment>
`;
